src/betterproto/plugin/main.py

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env python
-import sys
+
 import os
+import sys
 
 from google.protobuf.compiler import plugin_pb2 as plugin
 
-from betterproto.plugin.parser import generate_code
+from .parser import generate_code
```

Comment:
I'm not certain but I think switching to relative imports within the plugin might break invoking this module as a script which is otherwise supported.

Reply:
Seems like a good call

Reply:
It should be fine as tests seem to run right?

Reply:
I don't think there are any feature tests for that invoke the plugin in that way. It's generally not the most convenient way to do it but still worth supporting.